xla/CMakeLists.txt

```diff
@@ -1,10 +1,19 @@
-set(XLA_SOURCES jfdctxla.c)
-add_library(xla OBJECT ${XLA_SOURCES})
+# External tensorflow library.
```

Comment:
Is this only conditionally included?  It should be.

Reply:
This file itself is conditionally included. Look at [this](https://github.com/googleinterns/libjpeg-turbo/blob/xla/CMakeLists.txt#L548) line.